Src/CSharpier.Tests/TestFiles/TypeParameterConstraintClause/TypeParameterConstraintClauses.cst

```diff
@@ -12,16 +12,34 @@ class ClassName<T> where T : class
 
     public static ReturnType<T> MethodName<T, U>()
         where T : class
-        where U : class { }
+        where U : class
+    {
+        return;
+    }
 
     public static ReturnType<T> MethodName<T, U>(string parameter)
         where T : class
-        where U : class { }
+        where U : class
+    {
+        return;
+    }
 
     public static ReturnType<T> MethodName<T, U>(
-        string reallyLooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooongParameterIMeanIt
```

Comment:
It looks like you no longer have any tests for the "method with type constraints but no body" case.